codespell_lib/data/dictionary.txt

```diff
@@ -6946,6 +6948,7 @@ listensers->listeners
 listernes->listeners
 listner->listener
 listners->listeners
+liszt->list
```

Comment:
This is a name of a (somewhat?) famous composer, do you think we should disable it by default? Although this is "codespell" so I am a bit torn on this one, I imagine the false alarms will be much rarer than the true positives for this one

Reply:
you can always just do 
`liszt->list, liszt,`

Reply:
Thanks a lot @luzpaz . I'm going to upload new version